charts/karmada/templates/karmada-controller-manager.yaml

```diff
@@ -71,6 +71,10 @@ spec:
             initialDelaySeconds: 15
             periodSeconds: 15
             timeoutSeconds: 5
+          ports:
+            - containerPort: 8080
+              name: metrics
+              protocol: TCP
```

Comment:
hi, can you help explaining two question:

* If this `ports` field was not set, what will be the impact? Or saying which feature will not work?
* Whose port is port `8080` and do we listen on it in the karmada component?

Reply:
A1: If the port not set, the PodMonitor resource does not work. it's need to defined port name in the container.
A2: The 8080 is default port for metrics.